packages/java-parser/scripts/parse-samples.js

```diff
@@ -22,53 +22,105 @@ if (options.printErrors) {
   printErrors = console.error;
 }
 
-// const samplesDir = path.resolve(__dirname, "../samples/java-design-patterns");
-const samplesDir = path.resolve(__dirname, "../samples/spring-boot");
-const sampleFiles = klawSync(samplesDir, { nodir: true });
-const javaSampleFiles = sampleFiles.filter(fileDesc =>
-  fileDesc.path.endsWith(".java")
+const samplesDir = [];
```

Comment:
you need to update the sample spec
- https://github.com/jhipster/prettier-java/blob/master/packages/java-parser/test/samples-spec.js

Not add every single sample here directly.
This just a dev tool, e.g for local benchmarking